src/htwgnlp/__init__.py

```diff
@@ -0,0 +1 @@
+""""This is an empty __init__.py file to make Python treat the directory as a package."""
```

Comment:
If we provide a docstring, it should describe the contents of the `htwgnlp` package